app/src/main/java/com/anikinkirill/tinkoffsiriusmobile/MainActivity.java

```diff
@@ -1,15 +1,32 @@
 package com.anikinkirill.tinkoffsiriusmobile;
 
-import androidx.appcompat.app.AppCompatActivity;
-
+import android.content.Intent;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
 
 public class MainActivity extends AppCompatActivity {
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
-        System.out.println("Hello ");
     }
+
+    public void sn(View v){
```

Comment:
Maybe we gotta have more **_comprehendable_** names of methods?
For example, **'_sn_'**. As for me, not that clear